examples/examples/fetch_staking_details.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is part of subxt.
+//
+// subxt is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// subxt is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with subxt.  If not, see <http://www.gnu.org/licenses/>.
+
+//! To run this example, a local polkadot node should be running. Example verified against polkadot 0.9.13-d96d3bea85-aarch64-macos.
+//!
+//! E.g.
+//! ```bash
+//! curl "https://github.com/paritytech/polkadot/releases/download/v0.9.13/polkadot" --output /usr/local/bin/polkadot --location
+//! polkadot --dev --tmp
+//! ```
+
+use sp_keyring::AccountKeyring;
+use subxt::{
+    sp_core::{
+        sr25519,
+        Pair,
+    },
+    sp_runtime::AccountId32,
+    ClientBuilder,
+    DefaultConfig,
+    DefaultExtra,
+};
+
+#[subxt::subxt(runtime_metadata_path = "examples/polkadot_metadata.scale")]
+pub mod polkadot {}
+
+#[async_std::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    env_logger::init();
+
+    let api = ClientBuilder::new()
+        .build()
+        .await?
+        .to_runtime_api::<polkadot::RuntimeApi<DefaultConfig, DefaultExtra<DefaultConfig>>>();
+
+    let era = api.storage().staking().active_era(None).await?.unwrap();
+    println!(
+        "Staking active era: index: {:?}, start: {:?}",
+        era.index, era.start
+    );
+
+    let alice_id = AccountKeyring::Alice.to_account_id();
+    println!("  Alice account id:        {:?}", alice_id);
+
+    // Get Alice' Stash account ID
+    //
+    // *Note*: This is pre-funded, as declared in the [genesis state]
+    // (https://github.com/substrate-developer-hub/substrate-node-template/blob/main/node/src/chain_spec.rs#L49)
```

Comment:
Because it's polkadot and not substrate, I think the link would point to something more like:

https://github.com/paritytech/polkadot/blob/dc784f9b47e4681897cfd477b4f0760330875a87/node/service/src/chain_spec.rs#L1277

But in other examples we've taken for granted that some test accounts have funds in, so I wouldn't bother with a comment about it myself :)

Reply:
Thanks, i ve removed it for now :D